content/hi/docs/contribute/localization.md

```diff
@@ -0,0 +1,437 @@
+---
+title: कुबेरनेट्स प्रलेखन का स्थानीयकरण
+content_type: concept
+approvers:
+- divya-mohan0209
+- Garima-Negi
+weight: 50
+card:
+  name: contribute
+  weight: 50
+  title: डॉक्स का अनुवाद
+---
+
+<!-- overview -->
+
+यह पृष्ठ आपको दिखाता है कि कैसे विभिन्न
+[localize](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/) भाषाओं मे
+दस्तावेज़ (Docs) करे |
+
+
+<!-- body -->
+
+## मौजूदा स्थानीयकरण में योगदान करें
+
+आप मौजूदा स्थानीयकरण कंटेंट में लिखने या सुधारने में मदद कर सकते हैं। [कुबेरनेट्स
+स्लैक](https://slack.k8s.io/) में आपको प्रत्येक स्थानीयकरण के लिए एक चैनल मिलेगा| यहाँ एक सामान्य
+[SIG डॉक्स स्थानीयकरण स्लैक चैनल](https://kubernetes.slack.com/messages/sig-docs-localizations) भी है जहाँ आप हेलो कह सकते हैं।
+
+{{< note >}}
+यदि आप किसी ऐसे स्थानीयकरण पर काम करना चाहते हैं जो पहले से मौजूद है, तो जाँच करें
+यह पृष्ठ उस स्थानीयकरण में (यदि यह मौजूद है), के बजाय
+अंग्रेजी मूल। आप वहां अतिरिक्त विवरण देख सकते हैं।
+{{< /note >}}
+
+### अपना दो-अक्षर का भाषा कोड खोजें
+अपने स्थानीयकरण का दो-अक्षर वाला भाषा कोड खोजने के लिए सबसे पहले, [ISO 639-1 स्टैण्डर्ड ](https://www.loc.gov/standards/iso639-2/php/code_list.php)] से परामर्श करें । उदाहरण के लिए, कोरियाई के लिए दो-अक्षर का कोड
+`ko` है।
+
+### रिपॉजिटरी को फोर्क और क्लोन करे
+
+सबसे पहले, रिपोजिटरी में [अपना खुद का फ़ोर्क बनाएं](/docs/contribute/new-content/open-a-pr/#fork-the-repo) 
```

Comment:
I assume you want to say that fork the Kubernetes/website repository and not make a fork in the repository. Could you please modify to reflect the translation correctly?